openfl/component/assigner/static_grouped_assigner.py

```diff
@@ -85,10 +87,10 @@ def define_task_assignments(self):
                     # that task
                     self.collaborators_for_task[task][round_] += group_col_list
 
-    def get_tasks_for_collaborator(self, collaborator_name, round_number):
+    def get_tasks_for_collaborator(self, collaborator_name: str, round_number: int) -> Dict:
         """Get tasks for the collaborator specified."""
         return self.collaborator_tasks[collaborator_name][round_number]
 
-    def get_collaborators_for_task(self, task_name, round_number):
+    def get_collaborators_for_task(self, task_name: str, round_number: int) -> Dict:
```

Comment:
I guess:
```suggestion
    def get_collaborators_for_task(self, task_name: str, round_number: int) -> List[Any]:
```
May be  we should return `-> List[str]` in these cases?

Reply:
I'm not sure, I couldn't find confirmation for `List[str]`

Reply:
cc:/ @aleksandr-mokrov could you please clarify this?